Sources/PostgresNIO/Connection/PostgresConnection.swift

```diff
@@ -309,7 +309,7 @@ public final class PostgresConnection {
     private func queryStream(_ query: PostgresQuery, logger: Logger) -> EventLoopFuture<PSQLRowStream> {
         var logger = logger
         logger[postgresMetadataKey: .connectionID] = "\(self.id)"
-        guard query.binds.count <= Int(Int16.max) else {
+        guard query.binds.count < Int(UInt16.max) else {
```

Comment:
`UInt16.max` is `65535`. Given that `65535` are allowed, shouldn't this be `<=`?

Reply:
Correct, I forgot everything starts from 0 😅